### PR TITLE
rewrite error comparing

### PR DESCRIPTION
**Pull request by lm-kajihara**
related to #1107

### CODE REVIEW

File: blueoil/converter/templates/include/dlk_test.h [hunk 1]
```diff
@@ -38,22 +38,22 @@ struct Diff {
 
 namespace dlk_test
 {
-  static const T_FLOAT tolerance = 0.00001;
+  template<class T>static constexpr T tolerance = 0.00001;
 
   template<class T_IN, class T_RES>
-  inline bool same(T_IN input, T_RES result, T_IN& diff)
```
Comment (primenumber) [inline blueoil/converter/templates/include/dlk_test.h:44]: exact comparison may be needed for some types (integer, bool, etc...)

**Review by primenumber — APPROVED** [review]
RA

**Review by iizukak — APPROVED** [review]
OA

**Pull request merged**